lxd/devices.go

```diff
@@ -431,6 +431,22 @@ func getNumaNodeToCPUMap() (numaNodeToCPU map[int64][]int64, err error) {
 	return numaNodeToCPU, err
 }
 
+func getNumaCPUs(numaNodeToCPU map[int64][]int64, cpuNodes string) ([]int64, error) {
```

Comment:
missing doc block on this function.

As it would be useful to the caller and reviewer to understand what is in the returned values and what format the cpuNodes string is in.